frontend/src/index.tsx

```diff
@@ -39,6 +40,14 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                 path: AppRoute.SIGN_UP,
                 element: <Auth />,
               },
+              {
+                path: AppRoute.SURVEY,
```

Comment:
```suggestion
                path: AppRoute.SIGN_UP_SURVEY,
```
maybe it is better to rename for consistency with the backend side

Reply:
I left the previous naming because of the problem below